src/NRules/Tests/NRules.Json.Tests/NRules.Json.Tests.csproj

```diff
@@ -16,6 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\NRules.Fluent\NRules.Fluent.csproj" />
```

Comment:
@gritcsenko looking more at these tests, it bothers me that we are adding a dependency on NRules.Fluent to the NRules.Json-related project (even though it's just a test project). The idea is that the NRules.Json only depends on the NRules.RuleModel.
So, the way to test the serialization of the block expressions is via the ExpressionSerializerTest.

Reply:
@gritcsenko let me give you some more color on why I'm against that dependency on Fluent, and why serialization of Yield action may be more complicated.
Right now, Yield action is basically a syntactical sugar, that generates the block expression behind the scenes. However, I have always been thinking that eventually, Yield action will be added as the first-class citizen to NRules core and then serializing it will be fairly trivial.
With your changes it's technically possible to serialize it as a block expression, but in all honesty, if you look at that expression in json, it's pretty ugly and unwieldy, so has a pretty questionable use.
I still think that adding support for block expressions serialization is great, but this should hopefully make it clearer why I want to test it as a block expression, rather than as the Yield statement.

Reply:
I see. I need to modify tests. Thanks

Reply:
I was trying to replicate issue in test to be fair. Actual usecases can include both Fluent and Json libraries.

Reply:
@gritcsenko as you undoubtedly noticed, all fluent DSL does is generates the expression trees behind the scenes using RuleBuilder. So there is nothing that fluent does that we cannot setup in the test using builder directly. I’m looking at NRules.Json as just another DSL, so don’t want it to be a layer on top of fluent, but a sibling to fluent, only depending on the model.

Reply:
But I understand this test can be helpful as a temporary scaffolding to quickly loop on your changes

Reply:
> @gritcsenko as you undoubtedly noticed, all fluent DSL does is generates the expression trees behind the scenes using RuleBuilder. So there is nothing that fluent does that we cannot setup in the test using builder directly. I’m looking at NRules.Json as just another DSL, so don’t want it to be a layer on top of fluent, but a sibling to fluent, only depending on the model.

@snikolayev I can add architectural tests that will validate references between projects and classes, if you wish

Reply:
@gritcsenko maybe a good idea to add architectural tests at some point. Let's not do it under this PR though. For now, it's just a handful of projects in the solution, so may be an overkill